common/math/integer.go

```diff
@@ -51,6 +51,35 @@ func (i HexOrDecimal64) MarshalText() ([]byte, error) {
 	return []byte(fmt.Sprintf("%#x", uint64(i))), nil
 }
 
+// QuotedDecimal64 marshals uint64 as numeric value wrapped by a string.
+type QuotedDecimal64 uint64
```

Comment:
This is identical to `HexOrDecimal64`, except that on output, it renders as Decimal rather than Hex. But it still accepts Hex on input. 

How about naming it `Decimal64`? 

But I don't know. I don't quite see the benefit of adding this type, personally.

Reply:
Updated the name 👍 

> But I don't know. I don't quite see the benefit of adding this type, personally.

Why not add it though? The hex is not easily understandable. None of the gas costs are powers of two, so you don't see any nice relations to hex. I can say that I have spent a fair bit of time copying numbers into a python interpreter just to figure out what the diff in value between two traces are (usually missing a cold access cost somewhere).

The tooling won't care if it's decimal or hex, but on the occasion where we need to look at a trace by hand, this is very useful.